tests/diagnostics/array-zero-size.slang

```diff
@@ -2,12 +2,12 @@
 
 // Test that array size cannot be zero
 
-//TEST:SIMPLE:
+//DIAGNOSTIC_TEST:SIMPLE:
 
 [numthreads(4, 1, 1)]
 void computeMain(uint3 dispatchThreadID : SV_DispatchThreadID)
 {
     bar();
 }
 
-func bar() -> int[0];  // expected-error 30025 "array size must be larger than zero." 
\ No newline at end of file
+func bar() -> int[0];
```

Comment:
Before this line, you can write:
```
//CHECK: ([[# @LINE+1]]): error 30025
```
To match for `(13): error 30025` in the output.